Maquetacion/index.html

```diff
@@ -91,5 +91,10 @@ <h2 class="title-card">Django</h2>
         </div>
       </div>
     </div>
+    <footer class="footer-bar">
+      <p class="footer-text">we are game changers</p>
+      <img class="logo-w" src="assets/wolox_logo_white.svg" alt="Wolox-white">
+      <p class="copy"><small>Copyright 2018. Wolox. All rights reserved</small></p>
```

Comment:
modify the font size instead of the small tag

Reply:
👍

Reply:
Delete the small tag